emails/payin_bankwire_failed.spt

```diff
@@ -1,13 +1,13 @@
-{{ _("The bank wire to your Liberapay wallet has failed") }}
+{{ _("The bank wire to your Liberapay wallet failed") }}
```

Comment:
"Has" is redundant as "failed" is already in the past.

Reply:
The present perfect simple expresses an action that is still going on or that **stopped recently**.

Reply:
What actually happens, and how do you want it written?